HelpSource/Guides/WritingHelp.schelp

```diff
@@ -6,37 +6,37 @@ related:: Reference/SCDocSyntax, Classes/SCDoc
 section:: Writing new help
 The simplest way is to look at an existing help file or class document, and read this document and link::Reference/SCDocSyntax::
 
-note:: The help files should use UTF-8 encoding! ::
```

Comment:
Is there are reason this note is removed? I don't know of reasons anyone would write documents not in UTF-8, but this note was maybe added for a reason...

Reply:
fixed this in latest commit